src/styles/app.scss

```diff
@@ -151,7 +151,7 @@ h1, h2, h3, h4, h5, h6 {
 
 h1 {
   font-size: 1.75em;
-  line-height: 2.65;
+  line-height: 1.35;
```

Comment:
Agree with @Nikomus - though am worried it might revert to user agent defaults if we don't set one. I think h1's have them.

Am happy to leave if that could be the case. Seems silly to waste time changing if this works.